tests/phpunit/tests/kses.php

```diff
@@ -1936,11 +1936,18 @@ public function filter_wp_kses_object_added_in_html_filter( $tags, $context ) {
 	 *
 	 * @ticket 61009
 	 *
+	 * @dataProvider data_html_containing_various_kinds_of_html_comments
+	 *
 	 * @param string $html_comment    HTML containing a comment; must not be a valid comment
 	 *                                but must be syntax which a browser interprets as a comment.
 	 * @param string $expected_output How `wp_kses()` ought to transform the comment.
 	 */
-	public function wp_kses_preserves_html_comments( $html_comment, $expected_output ) {
+	public function test_wp_kses_preserves_html_comments( $html_comment, $expected_output ) {
+		$this->assertFalse(
+			current_user_can( 'unfiltered_html' ),
+			'Test needs to verify behavior without "unfiltered_html" capability.'
+		);
```

Comment:
[wp_kses](https://developer.wordpress.org/reference/functions/wp_kses/) itself does not check user caps right? So this is not needed?

Reply:
I think that's right, but the test is somewhat useless if that capability is available. it was working properly already, but I didn't know for sure. maybe it's best to nix this part. thank you.